architecture/tube.go

```diff
@@ -213,3 +214,33 @@ func (tube *Tube) Bury(command *Command) {
 		command.Err = errors.New(NOT_FOUND)
 	}
 }
+
+func (tube *Tube) Kick(command *Command) {
+	amount := 0
+	bound, err := strconv.Atoi(command.Params["bound"])
+	if err != nil {
+		// handle non-integer number of jobs to kick
```

Comment:
Also I'm thinking there should be a way to configure the command parameter types in `protocol_config.go` and handle the conversion at the command parse time. Would you like to work on that? I think it should not be that hard.